XamarinCommunityToolkit/Actions/Snackbar/Snackbar.macos.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Threading.Tasks;
```

Comment:
Can we leave only one file for both macos and iOS?

Snackbar.macos.ios.cs

Reply:
they are similar, but they used different Snackbar: IOSSnackbar and MacOSSnackBar

Reply:
Yeah, I see. But I believe we can resolve it by 

```
#if __IOS__
// iOS-specific code
#elif
// macOS
#endif
```